.github/workflows/deploy_web_nightly.yml

```
@@ -27,3 +31,28 @@ jobs:
+    runs-on: ubuntu-latest
+    if: github.event.repository.full_name == 'reearth/reearth-visualizer'
+    steps:
+      - uses: google-github-actions/auth@v0
+        with:
+          credentials_json: ${{ secrets.GCP_SA_KEY }}
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@v0
+      - name: Configure docker
+        run: gcloud auth configure-docker us-central1-docker.pkg.dev --quiet
+      - name: docker push
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:13: Double quote to prevent globbing and word splitting

(shellcheck)